docs/generic-skills.md

```diff
@@ -174,14 +174,14 @@ You can change the endpoint's address and port by modifying the connection's yam
 
 Under config locate :
 
-```bash
+``` bash
 addr: ${OEF_ADDR: 127.0.0.1}
 ```
  and replace it with your ip (The ip of the machine that runs the oef image.)
 
 Run both AEAs from their respective terminals
 
-```bash 
+``` bash 
```

Comment:
why the space?

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting

Reply:
We had some code with the space, so now its consistent. Works both ways..